twote/slackbot.py

```diff
@@ -0,0 +1,105 @@
+from slackclient import SlackClient
+import os
+from retweetbot import RetweetBot
+import time
+
+
+topic_list = ('#python,#pycon,#portland,#pyconopenspaces,#pycon2017,#pycon2016,#pythonic' +
+                   '#jobs,#career,#techwomen,' +
+                   '#angularjs,#reactjs,#framework,#pinax,#security,#pentest,#bug,#programming,#bot,#robot,' +
+                   '#calagator,#pdxevents,#events,#portlandevents,#techevents,' +
+                   '#r,#matlab,#octave,#javascript,#ruby,#rubyonrails,#django,#java,#clojure,#nodejs,#lisp,#golang,' +
+                   '#science,#astronomy,#math,#physics,#chemistry,#biology,#medicine,#statistics,#computerscience,#complexity,' +
+                   '#informationtheory,#knowledge,#philosophy,#space,#nasa,' +
+                   '#social,#economics,#prosocial,#peaceandcookies,#humility,#shoutout,' +
+                   '#opendata,#openscience,#openai,#opensource,' +
+                   '#data,#dataviz,#d3js,#datascience,#machinelearning,#ai,#neuralnet,#deeplearning,#iot,' +
+                   '#hack,#hacking,#hackathon,#compsci,#coding,#coder,' +
+                   '#mondaymotivation,#motivation,#life,#mind,' +
+                   '#play,#game,#logic,#gametheory,#winning,' +
+                   '#kind,#bekind,#hope,#nice,#polite,#peace,#inspired,#motivated,#inspiration,#inspiring,#quote,' +
+                   '#awesome,#beawesome,#payitforward,#give,#giving,#giveandtake,#love,#pause,#quiet,' +
+                   '#windows,#linux,#ubuntu,#osx,#android,#ios,' +
+                   '#thankful,#gratitude,#healthy,#yoga,#positivity,#community,#ecosystem,#planet,#meditation,#bliss,'
+                   ).split(',')
+
+
+_message = {
+    "text": "This is the tweet!",
+    "attachments": [
+        {
+            "text": "Retweet?",
+            "fallback": "You are unable to choose a game",
```

Comment:
oh yeah! I like to get something working and then cleaning it up :). Suggest changes.